src/Worker/HttpWorker.php

```diff
@@ -108,6 +108,7 @@ public function start(): void
 
         while ($request = $this->httpFoundationWorker->waitRequest()) {
             if ($this->shouldRedeclareTrustedProxies) {
+                \assert(0 <= $this->trustedHeaderSet && 64 > $this->trustedHeaderSet);
```

Comment:
Why this change? Is it a copy paste from Symfony?

Reply:
Would it pass if we type the prop like [this](https://github.com/symfony/symfony/blob/ecb9728f8d1deb559e9d8facbe0cdae62016200e/src/Symfony/Component/HttpFoundation/Request.php#L537)?